### DISCUSSION TITLE
Response headers issue

### DISCUSSION THREAD

**Discussion by Elendiar** [reduxjs/redux-toolkit#2999] [Q&A] [answered]
Hi, with `responseHandler` i always get only 2 headers in `response.headers`: *Content-Language* and *Content-Type*. On server side i send in response my custom header(app version) but cant get it on client. With postman i see at least 13 headers, but rtk query show me only 2. Why ?

**Comment by phryneas** [comment, marked as answer]
Probably your CORS settings do not allow any other response headers to be exposed to JavaScript. This is a server-side problem.

**Reply by Elendiar** [reply]
Oh thanks, adding in **django** settings (*django-cors-headers* installed) `CORS_EXPOSE_HEADERS = ['my-header-name']` fix this!
